.github/CONTRIBUTING.md

```diff
@@ -126,12 +140,48 @@ This wouldn't preserve the return value of the parent proc, instead, you should
 `.` is the current return value of the current proc.
 If you have the parent call at the end of the proc, use `return ..()` instead, it's slightly faster and more readable.
 
+### All `process` procs need to make use of delta-time and be frame independent
```

Comment:
i don't know if we actually have delta time implemented

Reply:
We don't